### PR TITLE
Blob/Base64 tidy up

### PR DESCRIPTION
**Pull request by stephencookdev**
# Problem
Currently our code surrounding blobs and base64 is a bit messy. We often convert to base64 when we don't need to, and convert things to canvas etc. when it's not needed

# Solution
This PR is split into 2 commits.

**The first commit** merges `file` and `canvas` into 1 new util file, `blob`. This file exports `canvasToBlob`, and `blobToLossyBase64` (as well as the previously existing `fileType` and `isTypeOf`).

Next, we remove any port to base64 out to the edges of where we're using it - so that we only call it when it's actually needed.

**The second commit** tidies updates the `Confirm` page, and `CaptureViewer` component to allow the base64 to be created at time of need (rather than being passed it up front)

It also tidies up the `Confirm` and `CaptureViewer` component in general, making the code and markup (hopefully) tidier, and a bit easier to read.

## Checklist

- [X] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [X] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Confirm/CaptureViewer/hocs.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    this.state = {
+      base64: null
+    }
+    this.updateBase64(blob)
+  }
+
+  updateBase64(blob) {
+    blobToLossyBase64(blob,
+      base64 => this.setState({ base64 }),
+      () => console.error("Uh oh"))
```
Comment (rfreitas) [inline src/components/Confirm/CaptureViewer/hocs.js]: Minor: we might have to make this a little more professional for customers :P

Comment (stephencookdev) [reply src/components/Confirm/CaptureViewer/hocs.js]: Ahh, I just put that there for testing. Thought I updated it, sorry 🙈

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-545-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://545-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**